tests/e2e/data/services/test_isolines.py

```diff
@@ -223,7 +221,8 @@ def test_isochrones_from_dataframe_as_new_table(self):
         self.assertTrue('data_range' in result)
         self.assertEqual(len(result.index), 6)
 
-        cdf = CartoDataFrame.from_carto(table_name, credentials=self.credentials)
+        cdf = CartoDataFrame.from_carto(table_name)
```

Comment:
Sorry, the Credentials instance is not required in the constructor, but it's required in the `from_carto` and `to_carto` methods.